spec/serializers/site_serializer_spec.rb

```diff
@@ -2,7 +2,27 @@
 
 RSpec.describe SiteSerializer do
   let(:site) { create :site }
+
   subject { serialize(site) }
 
-  it { is_expected.to include(name: site.location_name, campus_code: site.code, region_code: site.region_code) }
+  it { is_expected.to include(name: site.location_name, campus_code: site.code) }
+  it { is_expected.to include(region_code: site.region_code) }
+end
+
+
+RSpec.describe SiteSerializer do
```

Comment:
@misaka can you help here? I don't think we want 2 describe blocks

Reply:
Nest this describe into the above one ... 

```
RSpec.describe SiteSerializer do
  ...

  describe 'SiteSerializer#region_code' do
    ...
  end
end
```